apps/agent/src/mastra/workflows/github-notify-developer-about-pr-status.ts

```diff
@@ -60,6 +62,89 @@ async function handleWaitingForReview(
   }
 }
 
+async function handleUnconventionalTitleOrDescription(
+  discordUserId: string,
+  prs: PullRequest[],
+) {
+  const readyToCheckPRs = prs.filter((pr: PullRequest) => !pr.isWIP)
+  const titleFormatRegex = /^([a-zA-Z]+)(\([\w-]+(,[\w-]+)*\))?\:\s(.+)$/
+
+  // Check all PR descriptions in one batch
+  let prsNeedingDescriptionImprovement: PullRequest[] = []
+
+  if (readyToCheckPRs.length > 0) {
+    try {
+      const agentResponse = await suggestPRDescriptionAgent.generate([
+        {
+          role: 'user',
+          content: JSON.stringify(
+            readyToCheckPRs.map((pr) => ({
+              url: pr.url,
+              title: pr.title,
+              body: pr.body,
+            })),
+          ),
+        },
+      ])
+
+      try {
+        const prsNeedingImprovement = JSON.parse(agentResponse.text) as string[]
+        if (Array.isArray(prsNeedingImprovement)) {
+          // test
+          prsNeedingImprovement.push(
+            'https://github.com/dwarvesf/github-agent/pull/12',
+          )
```

Comment:
Looks like this should be removed

Reply:
sorry it was leftover from my local tests